api/src/main/proto/stellarstation/api/v1/groundstation/groundstation.proto

```diff
@@ -118,6 +118,20 @@ message Plan {
   // Configuration of the radio device used for uplinking to the satellite. Ground stations will
   // need to configure transmission during the plan to match this device.
   radio.RadioDeviceConfiguration uplink_radio_device = 6;
+
+  // The start of the time window reserved for the plan.
+  //
+  // This timestamp is always before aos_time and should be used by the ground station to prepare
+  // for the upcoming plan. Unlike aos_time, this timestamp does not change after the plan is
+  // reserved.
+  google.protobuf.Timestamp start_time = 7;
```

Comment:
Put these new fields directly above/below aos_time and los_time. I think the new fields should come first because they are the single representative time of a plan now. For the same reason comment for aos_time and los_time should be in relation to start_time and end_time, not the other way around.